data/helpers/recipeHelper.js

```diff
@@ -0,0 +1,15 @@
+const knex = require("knex");
+const knexConfig = require("../../knexfile");
+const db = knex(knexConfig.development);
+module.exports = {
+  getRecipes,
+  addRecipe
+};
+function getRecipes() {
+  return db("recipies");
```

Comment:
The recipes were supposed to also return of dishes they belong to, which means You need to use some `join` with the tables to get proper results.